reo/src/data_manager.py

```diff
@@ -431,16 +431,18 @@ def _get_REopt_cost_curve(self, techs):
                         itc_unit_basis = (tmp_cap_cost_slope[s] + rebate_federal) / (1 - itc)
 
                     sf = self.site.financial
-                    updated_slope = setup_capital_cost_incentive(itc_unit_basis,  # input tech cost with incentives, but no ITC
-                                                                 0,
-                                                                 sf.analysis_years,
-                                                                 sf.owner_discount_pct,
-                                                                 sf.owner_tax_pct,
-                                                                 itc,
-                                                                 eval('self.' + tech + '.incentives.macrs_schedule'),
-                                                                 eval('self.' + tech + '.incentives.macrs_bonus_pct'),
-                                                                 eval('self.' + tech + '.incentives.macrs_itc_reduction'))
-
+                    updated_slope = setup_capital_cost_incentive(
+                        itc_basis=itc_unit_basis,  # input tech cost with incentives, but no ITC
+                        replacement_cost=0,
+                        replacement_year=sf.analysis_years,
+                        discount_rate=sf.owner_discount_pct,
+                        tax_rate=sf.owner_tax_pct,
+                        itc=itc,
+                        macrs_schedule=eval('self.' + tech + '.incentives.macrs_schedule'),
+                        macrs_bonus_pct=eval('self.' + tech + '.incentives.macrs_bonus_pct'),
+                        macrs_itc_reduction=eval('self.' + tech + '.incentives.macrs_itc_reduction')
+                    )
+                    updated_slope *= two_party_factor
```

Comment:
@jmpohl let's hold on two party integration until Desktop validation. I know that this line is a bug that has already been fixed in the private repo (for example).